### ISSUE TITLE
I reject to having my hard work stolen by people

### ISSUE THREAD

**Issue by EggFriedCheese** [Blogshot/trip-sitter#10]
I would HATE it, if you gave someone an opportunity to steal my hard work that I put hours in to. This is NOT what the community wants. I guarantee that people will convert back, then forth, to then be able to upload their own version of the songs, that are identical to the original submitters. There is a reason .synth files are encrypted... don't abuse that.

**Comment by Blogshot** [comment]
Hey there,

as much as I understand your sentiment, I have a few objections.

- Many custom songs for SynthRiders were converted from BeatSaber. If you criticize this software, you therefore also need to criticize many of Synthrider's oldest custom songs
- It's hypocritical to distribute audio files without licensing or permission (which most `.synth` files do) and then complain about another party doing exactly that to your own work. "It's not as bad when I do it" 
- Personally, I believe that any art should be free to be transformed and republished, as long as the original author is credited (which this software does)
- At the time of programming this project, `.synth` files were not encrypted, meaning that even without this project SR's custom songs could be copied. As you can see it my code, it's only a matter of a few simple calculations that can easily be automated with excel
- Since `.synth` files were encrypted in the meantime, your whole argument is invalid. Because this software cannot decrypt these files it follows that anyone using this sotware has access to the original `.synth` and can be considered to have permission to convert it

I will gladly enter into a constructive discussion.

**Comment by EggFriedCheese** [comment]
There is no constructive discussion to be had here. I've put your comment to the mappers in the community, and NO-ONE is happy with this. 

`And then he brings up how hypocritical it is to complain about our work being shared because we're taking someone else's work (the songs themselves) and making something of it, but then it isn't hypocritical. In our case, the music certainly isn't ours, but the maps that can take many hours each are. They are ours. This converter doesn't have any of the person actually making content`

`Even if we're "stealing music for our art", at least we have art that said music goes into. This converter is inherently taking said art and doing nothing original with it`

There's less than 15% of all custom maps that were created by the convertor (that bearing in mind, is NOT a thing any more), and the community DO complain about converts, but what you are doing is stealing mappers' intellectual property bu doing

**Comment by Blogshot** [comment]
There is always a discussion to be had if viewpoints differ. I really hope that we can have a civil discussion that (de-)validates arguments instead of being emotional.

Firstly: I'd like to ask you to quote my replies directly to your community instead of summarizing them. This way people can read my original phrasing.

Secondly: I can understand how you feel about this project and that there is a deep dislike and distrust. I really do. However, I'm not convinced that your concerns are my responsibility (yet).

---

Your response covers my first two arguments, so I will address them.

##First paragraph:
Just as you're making something additional to your songs, the converter enabled people to create something new out of other's work. In the ReadMe, I explicitly state that the quality of the conversions is below the original and will need to be heavily modified in order to provide a fun experience; meaning that any conversion without transformative work will basically be a crippled version of the original map that is not fun to play. 

I will admit that malicious people were able to copy someone's work and only fix the maps instead of adding to it. However, this is not something you can fault the software for; the software is merely a tool. I can't control what people are using this tool for.

##Second paragraph:
Again, the intention of the software is to enable people to create transformative art. It's only natural that this converter won't add anything transformative by itself because it's just software. Of course it won't. It is expected that mappers add the creative part. That's the whole point.

Third paragraph:
You claim that I'm "stealing mappers' intellectual property". I have never published any conversion and most likely never will. In my understanding, I've never stolen anything.

---

My third argument is my personal opinion and not factual, so let's skip it.

I'd like you to answer to my 4th and 5th arguments. These are my main reasons for keeping the software available. **If you can oppose them, I'll consider deleting this project.**

Let me rephrase them as questions:

- Why was my software so problematic if a conversion can also be achieved with Excel? 
- Since `.synth` files have been encrypted for a long time now, in what way is my software actually usable to "plagiarize" other's work? Who, except the original mapper, is actually able to *use* this software in order to create conversions?

---

As a last note regarding the "hate":
You won't believe how little I care about hate or how big the community behind it is. Hate doesn't win discussions. Hate won't improve anyone's ability to persuade the opposing party.

For me, "hate" is just a childish symptom of people that either 
- are too emotional to rationally share their arguments, or 
- try to cover up that they didn't have good arguments to begin with

I don't see how I or my software is responsible for people abusing it. And as long as I don't break any law or you prove my points wrong, I don't see why I should care about your communities' opinion at all.

---

As I said before, I'm more than willing to continue this discussion in a factual manner.

**Comment by Nine3hundred** [comment]
@EggFriedCheese 
I would describe this less as people stealing your art and more as people collaborating with you on that art to improve on it and make it accessible to a bigger audience, it's pretty hyprocritcal to take other people's art as a basis for your art but get upset when someone takes your art as a basis for theirs.
